### ISSUE TITLE
readFileAssets block/freeze the UI

### ISSUE THREAD

**Issue by arshiyanaz** [itinance/react-native-fs#1220]
await readFileAssets('EstDataAA.xlsx', 'ascii').then((res) => {
 }).catch((err) => {

                const wb = XLSX.read(input(res), { type: 'binary' });
         const wsname = wb.SheetNames[0];
                const ws = wb.Sheets[wsname];
                const data = XLSX.utils.sheet_to_json(ws, { header: 1 });
            });

**Comment by birdofpreyru** [comment]
[#1197 &mdash; The future of **react-native-fs**](https://github.com/itinance/react-native-fs/issues/1197)

**TL;DR.:**
1. This repo is stale, no active work happening here now. I actively maintain a fork of the project, but it lives, and it is managed through a different repo &mdash; https://github.com/birdofpreyru/react-native-fs.
2. Such issue ticket is not actionable &mdash; it is not enough info to understand / reproduce the issue. Though, I guess, it is either your Excel file is too large to read through RN bridge with the current library implementation, or it might be that your post-processing of the promise is the issue.
3. In either case, it is unlikely somebody will act on the issue for free :)
